### DISCUSSION TITLE
Did a port on BL Forum about OrcaSlicer

### DISCUSSION THREAD

**Discussion by PuterPro** [OrcaSlicer/OrcaSlicer#503] [General]
@SoftFever - Just a little heads up for ya, I just did a PSA post plugging OrcaSlicer for you on the official Forum :)
Thanks for such a great piece of software!!
https://forum.bambulab.com/t/psa-up-your-game-studio-softfever-orcaslicer-arachne/7313

**Comment by SoftFever** [comment]
Nice write-up 👍
